package.json

```diff
@@ -17,15 +17,18 @@
   },
   "dependencies": {
     "brace": "^0.8.0",
+    "classnames": "^2.2.5",
```

Comment:
after reading stackoverflow answer and npmjs blog, I think it is because the maintainer use peerDep, because peerDep doesn't install peered dep when we run npm install, peerDep expect project which use 'react-flexbox-grid' to use babel-polyfill, classnames, flexboxgrid and react too.